projects/demo/src/modules/components/cell/index.ts

```diff
@@ -2,15 +2,26 @@ import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {TuiDemo} from '@demo/utils';
 import {tuiDocExampleOptionsProvider} from '@taiga-ui/addon-doc';
+import {TuiTitle} from '@taiga-ui/core';
+import {TuiAvatar} from '@taiga-ui/kit';
+import type {TuiCellOptions} from '@taiga-ui/layout';
+import {TuiCell} from '@taiga-ui/layout';
 
 @Component({
     standalone: true,
-    imports: [TuiDemo],
+    imports: [TuiAvatar, TuiCell, TuiDemo, TuiTitle],
     templateUrl: './index.html',
+    styleUrls: ['./index.less'],
     changeDetection,
     providers: [tuiDocExampleOptionsProvider({fullsize: true})],
 })
 export default class Example {
+    protected sizes = ['s', 'm', 'l'] as const;
+    protected size: TuiCellOptions['size'] = this.sizes[1];
+
+    protected heights = ['compat', 'spacious', 'default'] as const;
+    protected height: TuiCellOptions['height'] = this.heights[1];
```

Comment:
Should be `default` initial, let's make it first in the array as well

Reply:
fixed